cloud/cpp/SampleProducer.cc

```diff
@@ -17,20 +17,18 @@
  * under the License.
  */
 #include <iostream>
+#include <pulsar/Authentication.h>
 #include <pulsar/Client.h>
 
 using namespace pulsar;
 
-int main() {
+int main(int argc, char *argv[]) {
     ClientConfiguration config;
-    std::string params = R"({
-    "issuer_url": "https://dev-kt-aa9ne.us.auth0.com/oauth/token",
-    "private_key": "../../pulsar-broker/src/test/resources/authentication/token/cpp_credentials_file.json",
-    "audience": "https://dev-kt-aa9ne.us.auth0.com/api/v2/"})";
+    std::string params = argv[2];
```

Comment:
How about change `params` to `oauthParams`?

Reply:
Cool